bin/nodetool

```diff
@@ -126,6 +126,23 @@ do(Args) ->
                     io:format("RPC to ~p failed: ~p~n", [TargetNode, Reason]),
                     halt(1)
             end;
+        ["eval-elixir" | ListOfArgs] ->
+            JoinedArgs = string:join(ListOfArgs, " "),
+            case rpc:call(TargetNode, 'Elixir.Code', eval_string, [JoinedArgs]) of
+                {Value, []} ->
+                    %% Since we are in Erlang land, we ask Elixir to
+                    %% format the result as well:
+                    case rpc:call(TargetNode, 'Elixir.Kernel', inspect, [Value, [{pretty, true}]]) of
+                        {badrpc, Reason} ->
+                            io:format("RPC to ~p failed: ~p~n", [TargetNode, Reason]),
```

Comment:
Maybe we show `Value` as an Erlang term if _formatting_ on a remote node failed?

Reply:
I think it might be a little confusing if the output syntax is not constant, but it may be better than losing the result just because formatting failed, indeed!